woocommerce/woocommerce-ios
# [Woo POS] Update cart UI

Closes: #13262


## Description


Updated main parts of the Cart view
- removed checkout button when cart is empty
- updated fonts and colors
- updated Clear button
- added shadow

**Disclaimer**: some shadows, paddings and similar things might need tweaks later on when we update other parts of the POS UI

Related PR: https://github.com/woocommerce/woocommerce-ios/pull/13417 (make sure that one is merged first)

## Testing information


- Enter POS
- Check if the empty cart looks ok and there is not checkout button
- Add few products to the cart
- Check that the cart shows the products and checkout button is there
- Try removing products from the cart by tapping X button on them
- Try clearing the cart
- Add products again
- Tap checkout
- Finish the purchase

## Screenshots


![Simulator Screen Recording - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 38 03](https://github.com/user-attachments/assets/17deb0d5-b909-425b-a7ab-800494a3340a)

| Before      | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 44 02](https://github.com/user-attachments/assets/f50964b9-f16f-4d57-910d-f3171502f090)      | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 38 10](https://github.com/user-attachments/assets/53d0365f-1dff-400e-9d08-475aa30c50f4)       |
| ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 44 10](https://github.com/user-attachments/assets/b750d528-e627-48d5-aa95-8fd66d8f9705)      | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 38 25](https://github.com/user-attachments/assets/ab13c459-4a91-461b-9261-5a33f3f43192)       |
| ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 44 24](https://github.com/user-attachments/assets/80b9ae80-ecc3-46f4-b164-c972eac6b9b2)      | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 38 29](https://github.com/user-attachments/assets/152bf2c6-2857-4181-b1da-85494c794e47)       |

---
